hack/image-loader.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The KubeOne Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The image-loader script is used to pull all Docker images used by KubeOne,
+# Kubeadm, and Kubernetes, and push them to the the specified private registry.
+
+# WARNING: This script heavily depends on KubeOne and Kubernetes versions.
+# You must use the script coming the KubeOne release you've downloaded.
+
+set -euo pipefail
+
+KUBERNETES_VERSION=${KUBERNETES_VERSION}
```

Comment:
probably we need something better then this, otherwise it will blowup later.

Reply:
Do you have some idea about what should we do? Add some check at the beginning?

Reply:
yeah, make it required and if not provided — fail the whole script